src/main/kotlin/com/KotHub/models/KIssues.kt

```diff
@@ -0,0 +1,68 @@
+package com.KotHub.models
+
+import java.util.*
+
+data class KIssues(val id: Int?, val url: String?, val reporsitoryUrl: String?, val labelsUrl: String?, val commentsUrl: String?,
+                     val eventsUrl: String?, val htmlUrl: String?, val number: Int?, val state: String?, val title: String?, val user: KUser?,
+                     val labels: KLabels?, val assignee: KUser?, val assignees: Array<KUser>?, val milestone: KMilestone?,
+                     val locked: Boolean?, val comments: Int?, val pullRequests: KPullRequests?, val closedAt: String?,
+                     val createdAt: String?, val updatedAt: String?, val repository: KRepository?) {
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
Are the explicit implementations of `equals` and `hashCode` necessary?

According to the [Kotlin docs](https://kotlinlang.org/docs/reference/data-classes.html):

> The compiler automatically derives the following members from all properties declared in the primary constructor:
> - `equals()`/`hashCode()` pair;